src/hubmap_translation/addl_index_transformations/portal/tests/test_assay_details.py

```diff
@@ -279,6 +279,25 @@ def test_lab_processing():
     assert lab_processed_input_doc == lab_processed_output_doc
 
 
+def test_external_processing():
```

Comment:
This could be made into a parameterized test by combining with the two preceding test methods, `test_lab_processing` and `test_hubmap_processing`.

Reply:
@lchoy @john-conroy are we still waiting on any updates to address this comment or we are good to go?

Reply:
This can go as is; the tests can be modified later.